code/backtracking/src/print subaarrays with length.cpp

```diff
@@ -0,0 +1,44 @@
+#include<bits/stdc++.h>
+using namespace std;
+int l=0;
+void ans(int input[],int startindex,int size,int index,int output[],int osize,int k)
```

Comment:
Format your code according to our style guide, avoid global variables, and use `std::vector`.